pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1293,6 +1246,23 @@ func (p *PriorityQueue) calculateBackoffDuration(podInfo *framework.QueuedPodInf
 	return duration
 }
 
+// NominatedPodsForNode returns a copy of pods that are nominated to run on the given node,
+// but they are waiting for other pods to be removed from the node.
+// CAUTION: Make sure you don't call this function while taking any queue's lock in any scenario.
+func (p *PriorityQueue) NominatedPodsForNode(nodeName string) []*framework.PodInfo {
```

Comment:
nit: move this back to line 1247, so we can keep as much git history as possible.

Reply:
Done